### ISSUE TITLE
Vector assets processing fails

### ISSUE THREAD

**Issue by naixx** [alibaba/freeline#507]
Freeline fails in the following situation:
I have an animated-vector  defined in `drawable-v21` and fallback vector xml defined in `drawable`. Lets call it `my_vector`.
During clean build everything works fine. During incremental freeline compilation there is an error (editing databinding layout particularly)
```
freeline_id_keeper_public.xml:3025: error: Public symbol drawable/my_vector_1 declared here is not 
defined
freeline_id_keeper_public.xml:3025: error: Public symbol drawable/my_vector_2 declared here is not defined
```
Of course I don't define `my_vector_1` etc in the project. They are generated (by aapt?) in `app\build\intermediates\res\resources-debug.ap_\res\drawable-v21\` as binary xml. And of course they are not exposed in public

**Comment by lomanyong** [comment]
@naixx hi, thanks for your feedback. Could you provide a sample demo to replay this issue?

**Comment by lomanyong** [comment]
@naixx I just found out what caused this problem. The animated-vector generated the **_1 and **_2 ids and never really generated the target files so that aapt can not find the file.

**Comment by naixx** [comment]
Great, you've found it. Anything else you need from my side?

**Comment by lomanyong** [comment]
@naixx I'll fix this problem soon, thank.

**Comment by lomanyong** [comment]
@naixx fixed 0.8.5.

add your configuration in Freeline DSL:
````gradle
freeline {
    ...
    ignoreResourceIds = ["my_vector_1", "my_vector_2"]
}
````
